core/src/classes/configTemplate.ts

```diff
@@ -106,7 +107,7 @@ export abstract class ConfigTemplate {
   }
 
   formatId(s: string) {
-    return s.toLowerCase().replace(/[^a-zA-Z0-9-_\/.]/gi, "_");
+    return ConfigWriter.generateId(s);
```

Comment:
@evantahler Big difference here is that we _were_ allowing periods and slashes and not replacing them. But we have solid tests for the generate command, and they are still passing. IOW I was thinking that the slashes and period were unnecessary, and actually may cause and issue in some cases because we don't really want them in the ID values that we're generating.

Reply:
Sold!